master/internal/api_runs.go

```diff
@@ -158,7 +158,7 @@ func getRunsColumns(q *bun.SelectQuery) *bun.SelectQuery {
 			'progress', e.progress,
 			'forked_from', e.parent_id,
 			'external_experiment_id', e.external_experiment_id,
-			'is_multitrial', ((SELECT COUNT(*) FROM runs r WHERE e.id = r.experiment_id) > 1),
```

Comment:
Does this account for custom searchers? What about HP searches that only use one trial, are these marked with single or do we want to consider them multitrial? 

Is multitrial the concept we want to use moving forward? (This might be a more product/Model Dev question)

Reply:
From discussion: we want to use the searcher type. Custom searchers with one trial are still considered mult-trial here